app/assets/javascripts/app/project/edit.js

```diff
@@ -29,3 +35,40 @@ App.addChild('ProjectEdit', _.extend({
   }
 }, Skull.Tabs));
 
+// FIXME: MORE DRY HERE !!
```

Comment:
@diogob @adrianob i need some help here :/ I don't wanna repeat this code some suggestions?